### DISCUSSION TITLE
Live(?) transcription from the terminal

### DISCUSSION THREAD

**Discussion by JohnZolton** [openai/whisper#479] [Show and tell]
This is a psuedo-live transcription program using Whisper.

https://github.com/JohnZolton/scribe

A microphone recorder function and a whisper function run in parallel using multiprocessing. The microphone continuously records 5 second audio files while the whisper function checks if a new audio file is ready for transcription and then displays the transcription on the terminal

**Comment by Agusteando** [comment]
Hey John, I am trying to run your implementation but I am getting

python livemic.py
Traceback (most recent call last):
  File "C:\Users\desar\scribe\scribe\livemic.py", line 12, in <module>
    import whisper
  File "C:\ProgramData\Anaconda3\lib\site-packages\whisper.py", line 69, in <module>
    libc = ctypes.CDLL(libc_name, mode=ctypes.RTLD_GLOBAL)
  File "C:\ProgramData\Anaconda3\lib\ctypes\__init__.py", line 364, in __init__
    if '/' in name or '\\' in name:
TypeError: argument of type 'NoneType' is not iterable

**Reply by JohnZolton** [reply]
I'm sorry I have no idea how to fix that

**Reply by jongwook** [reply]
@Agusteando I think you're using a different package named whisper. Please try reinstalling the package by:

```bash
pip uninstall whisper
pip install git+https://github.com/openai/whisper.git 
```

**Reply by guyos46** [reply]
I had the same problem. @jongwook solution worked for me.

**Reply by samueldonovan1701** [reply]
I got this error when I had the wrong version of Python. Switching from 3.11 to 3.10 fixed it.
